app.rb

```diff
@@ -26,6 +22,10 @@ def menu
   end
 
   def run
+    # get_new = Option.new
```

Comment:
- [ ] Do not commit old pieces of code as inline comments. They will make your project look messy. If you need to review a previous version of your code, you can always use git history.